pkg/parsers/bitbucket/jobs.go

```diff
@@ -143,6 +141,17 @@ func parseExecutionUnitToStep(executionUnitRef *bitbucketModels.ExecutionUnitRef
 	return &step
 }
 
+func setStepType(step *models.Step) models.StepType {
+	if step.Shell != nil && step.Shell.Type != nil {
+		return models.StepType(*step.Shell.Type)
```

Comment:
I'm not sure we want to have the step type set to the type of the shell, but to use `models.ShellStepType` if it's a shell command step

Reply:
It's what we did before my changes, but I'll change it

Reply:
Maybe there is a special reason @danielciuraru86 did it that way